cfpb/cfgov-refresh
# Fix for linking video stills

Fix for linking video stills


## Changes

- Modified `cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html` to fix naming issue with boolean block.
- Modified `cfgov/v1/atomic_elements/organisms.py` to fix naming issue with boolean block. 

## Testing

- Visit the `http://localhost:8000/consumer-tools/everyone-has-a-story/` in Wagtail. 
- Click the `Link image and heading` checkbox 
- Preview the page
- Ensure the video and heading links to the appropriate pages.  

## Screenshots

<img width="1026" alt="screen shot 2017-04-24 at 3 51 58 pm" src="https://cloud.githubusercontent.com/assets/1696212/25355822/fd35615e-2905-11e7-8519-373b11764232.png">

## Notes

-

## Todos

-

## Checklist
